calico/felix/plugins/fiptgenerator.py

```diff
@@ -434,31 +434,22 @@ def filter_forward_chain(self, ip_version):
         # the raw table.
         forward_chain.append(
             "--append {chain} --jump ACCEPT --match mark "
-            "--mark {mark}/{mark}".format(
+            "--mark {mark}/{mark} "
+            "--match conntrack --ctstate UNTRACKED".format(
                 chain=CHAIN_FORWARD,
                 mark=self.IPTABLES_MARK_ACCEPT)
```

Comment:
This change is fine, but it will affect some future situations where a user has other funny things in their iptables - so I think it would be good to add a fuller comment to explain the thinking here.  This will allow a future developer to balance what you're thinking now against whatever future needs might arise.